beets/plugins.py

```diff
@@ -51,7 +52,7 @@ class BeetsPlugin(object):
     def __init__(self, name=None):
         """Perform one-time plugin setup.
         """
-        self.import_stages = []
+        self._import_stages = []
```

Comment:
It seems like we could (cheaply) avoid breaking compatibility by just changing the other name instead: for example, leave this import_stages and call the method get_import_stages or somesuch.